.vib/drupal/cypress/cypress/integration/drupal_spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows creating an article with an image', () => {
+  cy.login();
+  cy.visit('/node/add/article');
+  cy.fixture('articles').then((article) => {
+    cy.get('.text-full[name*=title]').type(`${article.newArticle.title} ${random}`);
+    cy.get('iframe.cke_wysiwyg_frame').iframe().find('p').type(`${article.newArticle.text}`);
```

Comment:
I'm concerned about the `get` selectors as they seem too specific, especially the second one. Sometimes there is now good option but we should always try to look for selectors that seem "stable" and that shouldn't be prone to change from version to version. Just as a suggestion, another option for the iframe selector would be:

```suggestion
    cy.get('.text-full[name*=title]').type(`${article.newArticle.title} ${random}`);
    cy.get('[title*="Body"]').iframe().find('p').type(`${article.newArticle.text}`);
```